### DISCUSSION TITLE
Question mark causes error on Windows Server 2016

### DISCUSSION THREAD

**Discussion by atakanguney** [laravel/framework#35181] [Q&A]
- Laravel Version: 8.0
- PHP Version: 7.3.18
- Database Driver & Version:

### Description:
I have created my authentication routes with Inertia stack by running following command

```php artisan jetstream:install inertia```

Everything works fine on my Macbook. But when I try to deploy my application on a Windows Server 2016(client's server). It is getting following error for authentication pages:

ErrorException
file_exists(): open_basedir restriction in effect. File(C:\inetpub\vhosts\example.com\resources\lang/en/Forgot your password?.php) is not within the allowed path(s): (C:/Inetpub/vhosts/example.com\;C:\Windows\Temp\;{C:\inetpub\vhosts\example.com\resources\lang}) (View: C:\inetpub\vhosts\example.com\resources\views\auth\login.blade.php)

If I remove question mark at the end of "Forgot your password?" in "login.blade.php", it works. I don't know if the issue is with Laravel or Windows. 

Note: In the same server, I am running another laravel application with version 7.x and it completely works fine.

### Steps To Reproduce:
- Deploy laravel 8.0 application with authentication pages to Windows Server 2016
- Load the "login" page

**Comment by bitPimps** [comment]
Just to add to this, this exact error also happens on Windows Server 2022 with Laravel 10
